Lucerne/MsEth1.xml

```diff
@@ -48,33 +49,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                            <material key="parchment"/>
                         </support>
                         <extent>
-                           <measure unit="leaf">12</measure>
-                           <note>dimensions for the folded book</note>
+                           <measure unit="leaf">3</measure>
+                           <measure unit="page">27</measure>
+                           <note>3 leaves, tied together and folded to 27 pages; dimensions for the folded book</note>
                            <dimensions type="outer" unit="mm">
                               <height>52</height>
                               <width>42</width>
-                              
                            </dimensions>
                         </extent>
-                        <foliation></foliation>
-                        <condition key="good"></condition>                        
+                        <foliation>No foliation</foliation>
+                        <collation>Three slender pieces of parchment have been tied together and folded into 27 pages, the first and the last  of which
```

Comment:
27 or 26? I counted 26

Reply:
The last page is glued to the back board. I do not know, how to count and how to make this point clear.